cvat/apps/dataset_manager/bindings.py

```diff
@@ -5,10 +5,10 @@
 
 from __future__ import annotations
 
+import logging
```

Comment:
Probably, you should use the task logger or the global server logger instead. Defined in `engine/log.py`.

Reply:
Agreed, thanks.